rhino/src/main/resources/org/mozilla/javascript/resources/Messages_zh_CN.properties

```diff
@@ -432,6 +432,9 @@ msg.no.paren =\
 msg.reserved.id =\
     标识符是保留字: {0}
 
+msg.super.shorthand.function =\
```

Comment:
<img width="539" alt="image" src="https://github.com/user-attachments/assets/cce713d7-dbed-4133-a143-ec3d4d79f4c1">

I'm not sure why, but this should be displayed like this. Probably my fonts don't have the correct character sets? :shrug: